commit 44dce7067d4cef681c2cfc26b2d9c06fd120bba4 (refs/changes/40/1240/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-01-12T11:39:09-08:00 (2 years, 9 months ago)
    
    joyent/node-triton#163 "skipAffinityTests" option for test/config.json is not documented in test/config.json.sample

diff --git a/test/config.json.sample b/test/config.json.sample
index 957fa18..44e2e65 100644
--- a/test/config.json.sample
+++ b/test/config.json.sample
@@ -9,12 +9,18 @@
         "account": "joe.blow",
         "keyId": "de:e7:73:32:b0:ab:31:cd:72:ef:9f:62:ca:58:a2:ec",
         "insecure": false
-    }
+    },
 
     // Optional. Set this to allow the parts of the test suite that create,
     // update and destroy resources: instances, images, networks, etc.
     // This is essentially the "safe" guard.
-    "allowWriteActions": true
+    "allowWriteActions": true,
+
+    // Optional. Set to 'true' to skip affinity tests. Affinity tests assume
+    // the datacenter has multiple servers to which provisions can go. So if
+    // you are testing against COAL, for example, you'll need to set this
+    // to true.
+    "skipAffinityTests": false,
 
     // The params used for test provisions. By default the tests use:
     // the smallest RAM package, the latest base* image.
